planner/core/testdata/plan_suite_out.json

```diff
@@ -963,31 +963,31 @@
       },
       {
         "SQL": "select sum(e), avg(e + c) from t where c = 1 group by e",
-        "Best": "IndexReader(Index(t.c_d_e)[[1,1]]->HashAgg)->HashAgg"
```

Comment:
Whether to push Agg down to TikV when small data size is a little unstable.
It will be resolved by https://github.com/pingcap/tidb/issues/38874.